drafts/SC1-2-2-captions-prerecorded.md

```diff
@@ -0,0 +1,171 @@
+# SC1-2-2-captions-prerecorded
+
+## Description
+This test checks that prerecorded synchronized media (almost video) contains captions, which are a complete substitute of the audio content.
+
+## Background
+- [Captions (Prerecorded): Understanding SC 1.2.2](http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-captions.html)
+* [G93: Providing open (always visible) captions](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G93)
+* [G87: Providing closed captions](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G87)
+* [F8: Failure of Success Criterion 1.2.2 due to captions omitting some dialogue or important sound effects](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F8)
+* [F75: Failure of Success Criterion 1.2.2 by providing synchronized media without captions when the synchronized media presents more information than is presented on the page](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F75)
+* [F74: Failure of Success Criterion 1.2.2 and 1.2.8 due to not labeling a synchronized media alternative to text as an alternative](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F74)
+
+## Assumptions
+- The user defines if this test case is applicable to content that is not a video but animated
```

Comment:
👍, can't we generally assume that basic understanding of WCAG is required for all rules?